x/cron/keeper/grpc_query_schedule_test.go

```diff
@@ -134,7 +134,7 @@ func createNSchedule(t *testing.T, ctx sdk.Context, k *cronkeeper.Keeper, n int3
 		item.Msgs = nil
 		item.LastExecuteHeight = uint64(ctx.BlockHeight())
 
-		err := k.AddSchedule(ctx, item.Name, item.Period, item.Msgs)
+		err := k.AddSchedule(ctx, item.Name, item.Period, item.Msgs, uint64(item.Blocker))
```

Comment:
Casting to uint64 looks a bit weird.
Can we just make AddSchedule last arg of `BlockerType` instead of uint64?

Reply:
Done